README.md

```diff
@@ -237,7 +234,7 @@ on GitHub or get a copy using git with the following command:
   * Support for mouse wheel zooming (by Dave Reisner)
   * Added fullscreen mode
 
-**[v0.1](https://github.com/muennich/sxiv/archive/v0.1.tar.gz)**
+**[v0.1](https://github.com/nsxiv/nsxiv/archive/v0.1.tar.gz)**
 *(January 21, 2011)*
```

Comment:
I think we should leave the original changelog as is. Maybe move it over to a seperate file, "CHANGELOG".

Reply:
Indeed, having a separate file would be nice. I think the README is too cluttered right now.

Reply:
Do what i did with rxiv, move the sxiv changelog to it's own file and only keep the changelog for nsxiv on the readme and update it upon release tags.

Reply:
> I think we should leave the original changelog as is

I disagree. I don't see any reason not to accept this change, it will keep working fine. Now that this repo is detached I think it makes even more sense to not link to the muennich repo in this case.

> Do what i did with rxiv, move the sxiv changelog to it's own file and only keep the changelog for nsxiv on the readme and update it upon release tags.

Not so sure if this is the best approach, since all previous versions should be available to both download and see the changelog.

The way I handled this on my fork ([changelog](https://github.com/GRFreire/sxiv/blob/master/changelog.md)) was to simply change all links to my fork repo and keep adding versions (not delete the old ones), eg: v27

Reply:
What I meant to highlight was this part https://github.com/nsxiv/nsxiv/pull/1/commits/1bf4e721ee86faeaa1bc76709273a0b569987419#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L112
I don't think we should change the original changelog. As for the links, I don't have any strong opinion on them.